vendor/github.com/blang/semver/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License
+
+Copyright (c) 2014 Benedikt Lang <github at benediktlang.de>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
```

Comment:
Out of caution - is this any concern since we are wrapping around this functionality since we have Apache license and not MIT? I'm not much of a copyright person - @joestringer may be able to provide feedback here?

Reply:
MIT is very liberal in that the only requirements are retaining license/copyright notice for the MIT-licensed code:

https://choosealicense.com/licenses/mit/

Reply:
Great, thanks!